client/src/components/App/stories/index.story.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+const renderLink = (url, text) => (
+  <a href={url} target="_blank" rel="noopener noreferrer">
+    {text}
+  </a>
+);
+
+storiesOf('Welcome Story', module)
+  .add('MERN Chatbot', () => (
+    <div style={{ padding: '25px' }}>
```

Comment:
Oh no! Add class for this :) It will be easy to write and do refactoring in the future.

Reply:
Good point, I was thinking about this but have not done yet. Thanks for the comment! 😃

Reply:
After starting to change inline styles to classes I remembered my main point why I decided to do not use classes in the very beginning. As the main idea of Storybook is to show how to use the components, I wanted to keep wrappers as transparent as possible, so inline styles are perfect for this purpose - you can see immediately all the styles that are applied to the container. It is true that in case if I decide to change the padding from 25 to 30 px someday it will be painful, so I just created a constant and changed all the values to be a constant.

Not sure if my reasoning is convincing enough for you, but I just want to keep it this way in my project. :)

Reply:
I understand you. I think your solution is good. Now it is more easy to edit code.
I would like to notice that now you can move to the constants all styles that are often used in storybook:
```
const paddingSize = '25px';
const defaultWrapperStyle = {
    padding: paddingSize,
};
```